samples/golang/mywebapi/src/app/hello.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
What's the reason for putting this file under src/app instead of directly in the mywebapi directory?

Reply:
There is no specific reason. Just following the structure format we have for the other samples (Java)

Reply:
That's a Java specific requirement.